create-container-image-job.yaml

```diff
@@ -3,17 +3,26 @@ apiVersion: batch/v1
 kind: Job
 metadata:
   name: load-container-image
+  namespace: quay-perf
 spec:
-  parallelism: 100
-  completions: 100
+  parallelism: ${PARALLELISM}
+  completions: ${PARALLELISM}
```

Comment:
IMHO lets have these be two different vars... `${PARALLELISM} and ${CONTAINER_COUNT}`